cirq-core/cirq/qis/clifford_tableau.py

```diff
@@ -583,6 +583,6 @@ def apply_global_phase(self, coefficient: linear_dict.Scalar):
         pass
 
     def measure(
-        self, axes: Sequence[int], seed: 'cirq.RANDOM_STATE_OR_SEED_LIKE' = None
+        self, axes: Sequence[int], seed: 'cirq.RANDOM_STATE_OR_SEED_LIKE'
```

Comment:
Are we sure that removing the default won't break anything?  Is it possible anyone is calling measure() without specifying a seed?  (Same for below).

Reply:
I've put the default back but set it to `np.random` instead of  `None` because it's needed by `_measure`.